CHANGELOG.md

```diff
@@ -9,6 +9,16 @@ x.y.z Release notes (yyyy-MM-dd)
   ([Core #7552](https://github.com/realm/realm-core/pull/7552)).
 * Improve perfomance of IN queries and chained OR equality queries for
   UUID/ObjectId types. ([.Net #3566](https://github.com/realm/realm-dotnet/issues/3566))
+  UUID/ObjectId types. ([.Net * #3566](https://github.com/realm/realm-dotnet/issues/3566))
+* Added support for updating Atlas Device Sync's base url, in case the need to roam between 
+  servers (cloud and/or edge server). This API is private and can only be imported using 
+  `@_spi(Private)` 
```

Comment:
SPI names are a global namespace, so this needs to be `RealmSwiftPrivate`. I'm also not sure why this is private, specifically? Is this *never* going to be a public part of the API for some reason? If it's just not GA then it should be something like `RealmSwiftUpcoming` or `RealmSwiftExperimental` or Unstable or something.

Reply:
Experimental is more like it